e2e-cypress/Jenkinsfile.template

```diff
@@ -1,9 +1,9 @@
 // See https://www.opendevstack.org/ods-documentation/ for usage and customization.
 
-@Library('ods-jenkins-shared-library@@ods_git_ref@') _
+@Library('ods-jenkins-shared-library@3.x') _
 
 odsComponentPipeline(
-  imageStreamTag: '@ods_namespace@/jenkins-slave-nodejs10-angular:@ods_image_tag@',
+  imageStreamTag: 'ods/jenkins-slave-nodejs10-angular:3.x',
```

Comment:
You need to leave the variables in - that should always be replaced when the quickstarter is provisioned with the versions that the user chose.

Reply:
Hi Michael, yes this was an accidental copy/paste change that I already fixed last Friday directly after my first commit (I guess that's why GitHub says "Outdated").

Reply:
Sorry, didn't realise. It showed that to me as part of the diff. Now if I go there it isn't there anymore ...

Reply:
Yeah, I guess GitHub send you an email with a direct link to the changes of the first commit. They should change that and make it a link to the changes of the full PR. :-)